Libraries/Text/TextInput/Multiline/RCTMultilineTextInputView.m

```diff
@@ -38,7 +38,7 @@ - (instancetype)initWithBridge:(RCTBridge *)bridge
     _backedTextInputView.scrollEnabled = YES;
 #else // [TODO(macOS ISS#2323203)
     _scrollView = [[RCTUIScrollView alloc] initWithFrame:self.bounds]; // TODO(macOS ISS#3536887)
-    _scrollView.backgroundColor = [UIColor clearColor];
+    _scrollView.backgroundColor = [RCTUIColor clearColor];
```

Comment:
Missing the TODO comment

Reply:
this change is already inside a ` // [TODO(macOS ISS#2323203)` ... ` // ]TODO(macOS ISS#2323203)` block.

Reply:
The line above it is marked specifically with a with a todo, makes it confusing if we're inconsistent

Reply:
There *are* inconsistencies.   The comment on line 40 for RCTUIScrollView is not necessary as this is inside a `[` ... `]` block